src/vstart.sh

```diff
@@ -1314,6 +1315,11 @@ else
         debug ms = 1'
 fi
 
+if [ "$ceph_osd" == "crimson-osd" ]; then
+    AUTOSCALER_OPTS='
+        osd_pool_default_pg_autoscale_mode = off'
+fi
+
```

Comment:
Not sure we need `src/` in the commit title. How about:
`vstart.sh: disable pg-autoscaling for crimson-osd`?